articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+
+What if you want to extend your site to have cool features that you don’t want to build from the scratch, GatsbyJS offers [plugins](https://www.gatsbyjs.com/plugins/) built by a community of developers who want to share cool and awesome features with everyone.
+
+We will be using a plugin for [Typogyaphy Js](https://kyleamathews.github.io/typography.js/) (a CSS framework) to style up the CSS by default.
+
+To use this plugin, you have to do this:
+
+```bash
+npm install gatsby-plugin-typography react-typography typography
+```
+
+When the installation is done go to the *gatsby-config.js* file, you want to tell Gatsby that you want to use this plugin.
+
+Add these lines of code in the *gatsby-config.js* file.
+
+``` JavaScript
+//plugin code
+module.exports = {
+plugins: [ `gatsby-plugin-typography`]
+}
+```
+
+### Building the site on a web server
+
+So far so good, your site has been running on your local server which is `http://localhost:8000`, what’s the use of building a static site without showing it to your friends, family, and probably your boss online.
+
+To get your Gatsby site ready for cloud deployment you will need to build it using this command:
+
+```bash
+npm run build
+```
+
+You will notice some changes in the public folder, that’s where your static site files are. If you want to host it on a webserver you could copy the public folder and place it on a server. Otherwise, you could use a cloud static site server like Amazon, Google, Netlify, Gatsby Cloud, Heruko, etc. to place the whole file directory on a cloud server.
+
+### Deploying to a cloud server
+
+You can deploy your Gatsby site online in a variety of ways, but there are 2 easy methods that we will discuss.  Those being [Netlify](https://www.netlify.com/) and [Gatsby Cloud](https://www.gatsbyjs.com/cloud/).
+
+**Netlify:** With Netlify you can drag and drop your code folder into it or deploy your site from [GitHub](https://github.com/) if your code is on GitHub, but I would recommend that you place your code on GitHub to automatically sync any changes made to the site.
+
+**Gatsby cloud:** This provides a simpler way to deploy the site, but it requires your code to be on GitHub or Gitlab. Deploying your site on Gatsby cloud offers you the flexibility to use content management systems like [Contentful](https://www.contentful.com/), [DatoCMS](https://www.datocms.com/), [Strapi](https://strapi.io/), and [Wordpress](https://wordpress.com/) etc.
+
+[Live site deployed on Netlify](https://my-first-gatsby-sites.netlify.app/)
+[Github code](https://github.com/Jethro-magaji/first-gatsby-site)
+[Live site deployed on Gatsby cloud](https://build-6a85bef9-47a2-4002-bb74-1b44f3dbf5bd.gtsb.io/)
+
+
+### Summary
+
+In summary, we covered the advantages JAMStack and GatsbyJS, you created your first GatsbyJS site, added content to it, linked between pages, learned about React components, made the site interactive with a counter, and lastly deployed it to the cloud.
```

Comment:
- End sentence after GatsbyJS. Start new at "You created..."
- Remove "with a counter", "lastly."

Reply:
Done